### DISCUSSION TITLE
space_and_separator_restorer.uc.js no longer working

### DISCUSSION THREAD

**Discussion by NoNameNeeded5** [Aris-t2/CustomJSforFx#118] [Q&A]
The script space_and_separator_restorer.uc.js is no longer working on Firefox 127+

**Comment by Aris-t2** [comment]
Are you sure?
![image](https://github.com/user-attachments/assets/ddc49f0b-2a2b-4f70-989a-5e77264f2403)
![image](https://github.com/user-attachments/assets/eadb8272-e51b-4375-8d85-b7e282bf1b9c)

**Comment by NoNameNeeded5** [comment]
Could it be the config.js file that I use?

That's the one I use (and the scripts run much faster than with other config.js files)

// skip 1st line
lockPref('xpinstall.signatures.required', false);
lockPref('extensions.install_origins.enabled', false);

try {
  const cmanifest = Cc['@mozilla.org/file/directory_service;1'].getService(Ci.nsIProperties).get('UChrm', Ci.nsIFile);
  cmanifest.append('utils');
  cmanifest.append('chrome.manifest');
  Components.manager.QueryInterface(Ci.nsIComponentRegistrar).autoRegister(cmanifest);

  const objRef = ChromeUtils.import('resource://gre/modules/addons/XPIDatabase.jsm');
  objRef.XPIDatabase.SIGNED_TYPES.delete("extension");

  Cu.import('chrome://userchromejs/content/BootstrapLoader.jsm');
} catch (ex) {};

try {
  Cu.import('chrome://userchromejs/content/userChrome.jsm');
} catch (ex) {};
